### PR TITLE
Store rawData on google_iam_bindings again

### PR DESCRIPTION
**Pull request by mknoedel**
This was originally taken out because it was assumed that this was causing a RequestTooLarge exception. This was not the issue and can thus be safely be re-added. Plus, the rawData for bindings is actually really small. Example:

```
      "_rawData": [
        {
          "name": "default",
          "rawData": {
            "role": "roles/cloudkms.serviceAgent",
            "members": [
              "serviceAccount:service-167984947943@gcp-sa-cloudkms.iam.gserviceaccount.com"
            ]
          }
        }
      ]
```

### CODE REVIEW

File: src/steps/compute/__snapshots__/index.test.ts.snap [hunk 1]
```diff
@@ -16621,7 +16621,7 @@ Object {
       "_class": Array [
```
Comment (mknoedel) [inline src/steps/compute/__snapshots__/index.test.ts.snap:16621]: unrelated change. Needed to update snapshot to get tests to pass.

**Review by austinkelleher — APPROVED** [review]

**Review by austinkelleher — APPROVED** [review]
